crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1732,12 +1739,14 @@ impl Room {
     #[instrument(skip_all)]
     pub fn send_attachment<'a>(
         &'a self,
-        body: &'a str,
+        caption: Option<String>,
+        formatted: Option<FormattedBody>,
+        url: &'a str,
```

Comment:
Why is that parameter named `url` when it is the filename?

Reply:
I thought it was clearer anyway :thinking: An url can be a filename, but a body can be anything.
I can change it if you think it's confusing

Reply:
You could just name it `filename`.